examples/NeuralNetwork-train-and-save/sketch.js

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2024 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+// This example will train a neural network to recognize hand gestures for rock paper scissors.
+
+let classifier;
+let handPose;
+let video;
+let hands = [];
+let rockDataCount = 0;
+let paperDataCount = 0;
+let scissorsDataCount = 0;
+let isTrained = false;
+let classification = "";
+
+//UI elements
+let instructionP;
+let dataCountsP;
+let rockButton;
+let paperButton;
+let scissorsButton;
+let trainButton;
+
+function preload() {
+  // Load the handPose model, we will use the keypoints form handPose to train the neural network
+  handPose = ml5.handPose();
+}
+
+function setup() {
+  createCanvas(640, 480);
+
+  // Create the webcam video and hide it
+  video = createCapture(VIDEO);
+  video.size(width, height);
+  video.hide();
+
+  // Setup the UI buttons for training
+  instructionP = createP(
+    'To add data, hold up hand gesture for rock while pressing the "Add Rock Data" button. Likewise for paper and scissors. When you have a good amount of data for each gesture class, press the "Train and Save Model" button to train the model and save it for future use.'
+  );
+  instructionP.style("width", "640px");
+  dataCountsP = createP(
+    "Rock data: " +
+      rockDataCount +
+      "<br>Paper data: " +
+      paperDataCount +
+      "<br>Scissors data: " +
+      scissorsDataCount
+  );
+  rockButton = createButton("Add Rock Data");
+  rockButton.mousePressed(addRockData);
+  paperButton = createButton("Add Paper Data");
+  paperButton.mousePressed(addPaperData);
+  scissorsButton = createButton("Add Scissors Data");
+  scissorsButton.mousePressed(addScissorsData);
+  trainButton = createButton("Train and Save Model");
+  trainButton.mousePressed(train);
+
+  // For this example to work across all browsers
+  // "webgl" or "cpu" needs to be set as the backend
+  ml5.setBackend("webgl");
+
+  // Set up the neural network
+  let classifierOptions = {
+    task: "classification",
+    debug: true,
+  };
+  classifier = ml5.neuralNetwork(classifierOptions);
+
+  // Start the handPose detection
+  handPose.detectStart(video, gotHands);
+}
+
+function draw() {
+  //Display the webcam video
```

Comment:
```suggestion
  // Display the webcam video
```

obsessive nitpicking!